analysis/soil/fungi_stoichiometry/sporocarp_CNP.R

```diff
@@ -0,0 +1,90 @@
+#| ---
+#| title: Calculate the C:N and C:P ratios of fungal fruiting body (sporocarp)
+#|
+#| description: |
+#|     This R script estimates the C, N and P mass fraction from a global
+#|     dataset and then estimates the C:N and C:P ratio of fruiting body.
+#|
+#| virtual_ecosystem_module: Soil
+#|
+#| author:
+#|   - Hao Ran Lai
+#|
+#| status: final
+#|
+#| input_files:
+#|   - name: Sporocarp+chemistry.xlsx
+#|     path: data/primary/soil/fungi_stoichiometry
+#|     description: |
+#|       Saprotrophic and ectomycorrhizal fungal sporocarp stoichiometry
+#|       across temperate rainforests by Kranabetter et al. (2018)
+#|       https://doi.org/10.1111/nph.15380; downloaded from
+#|       https://doi.org/10.5061/dryad.1d92k70
+#|
+#| output_files:
+#|   - name: CN_CP_ratio.csv
+#|     path: data/derived/soil/fungi_stoichiometry
+#|     description: |
+#|       Estimated C:N and C:P ratio for fungal fruiting body
+#|
+#| package_dependencies:
+#|     - tidyverse
+#|     - readxl
+#|     - glmmTMB
+#|
+#| usage_notes: |
+#|     The dataset covers only saprotrophic and EM fungi, but I think they are
+#|     good enough for us at this stage. There are not a lot of stoichiometry
+#|     specifically for the fruiting body in the literature.
```

Comment:
Might be worth noting that this data is temperate (which is unavoidable given the paucity of data) and that stoichiometry might well vary latitudinally

Reply:
Done!